Core/clim-core/commands.lisp

```diff
@@ -1059,49 +1056,54 @@ examine the type of the command menu item to see if it is
 
 (define-presentation-method accept
     ((type stream-destination) stream (view textual-view)
-     &key)
-  (let ((dest (eval (accept 'form
-			    :stream stream
-			    :view view
-			    :default *standard-output*))))
+                               &key
+                               (default '*standard-output*)
+                               (prompt "stream form"))
```

Comment:
maybe `stream-var` or `stream-symbol` or `stream-name`?

Reply:
The argument is evaluated so it really is a form.